src/main/java/org/wildfly/security/x500/X500.java

```diff
@@ -137,4 +137,32 @@ private X500() {}
             return Arrays.copyOf(certificates, certificates.length, X509Certificate[].class);
         }
     }
+
+    /**
+     * Convert an X.500 attribute name (AVA keyword) to OID
+     * @param attribute X.500 attribute name
+     * @return corresponding OID or {@code null} if was not recognized
```

Comment:
There is much more OIDs (http://www.alvestrand.no/objectid/2.5.4.html) , then handled by this method. Is it possible to leave input untransformed. I mean to add to switch something like `default: return attribute;` What do you think?